static/app/views/monitors/components/row.tsx

```diff
@@ -72,6 +72,13 @@ function MonitorRow({monitor, monitorEnv, organization, onDelete}: MonitorRowPro
     <TimeSince unitStyle="regular" date={monitorEnv.lastCheckIn} />
   ) : null;
 
+  const extraEnvironmentsWarning =
+    monitor.environments.length > 1
+      ? ` This monitor has checkin data associated with these environments which will be affected: ${monitor.environments
```

Comment:
```suggestion
      ? ` This will delete check-in data for this monitor associated with these environments: ${monitor.environments
```

Reply:
You don't have to copy the whole sentence, but `check-in` is hyphenated

Reply:
ahh yes this is nicer, also converting the list of environments to UL for better display